src/renderer/ui/components/modals/AlarmModal.js

```diff
@@ -81,6 +81,7 @@ class AlarmModal extends Component {
       >
         <TimePicker
           hintText={TranslationProvider.query('button-text-click-here-to-set-alarm')}
+          textFieldStyle={{width: '100%'}}
```

Comment:
http://www.material-ui.com/#/components/text-field

This should be replaced with the `fullWidth` property